t/app/controller/report_inspect.t

```diff
@@ -225,22 +225,27 @@ FixMyStreet::override_config {
         $report->discard_changes;
         my $comment = ($report->comments( undef, { order_by => { -desc => 'id' } } )->all)[1]->text;
         is $comment, "This is a public update.", 'Update was created';
+        is $report->get_extra_metadata('inspected'), 1, 'report marked as inspected';
         $mech->get_ok("/report/$report_id");
         my $meta = $mech->extract_update_metas;
         like $meta->[0], qr/State changed to: Action scheduled/, 'First update mentions action scheduled';
+        like $meta->[2], qr/Posted by .*defect raised/, 'Update mentions defect raised';
 
         $user->unset_extra_metadata('categories');
         $user->update;
     };
 
     subtest "test update is required when instructing" => sub {
+        $report->unset_extra_metadata('inspected');
         $report->update;
+        $report->inspection_log_entry->delete;
         $report->comments->delete_all;
         $mech->get_ok("/report/$report_id");
         $mech->submit_form_ok({ button => 'save', with_fields => { public_update => undef, include_update => "1" } });
```

Comment:
Should this line have raise_defect => 1 in it? Otherwise it doesn't appear to be testing it.

Reply:
Have added a separate test to check that public update is required when raising a defect. I've also given this test a clearer title, because it doesn't actually have anything to do with instructing.